README.md

```diff
@@ -63,42 +61,121 @@ yarn run start
+* Apache license
 
+The id is used by docusaurus to link the page to the specific point in the documentation menu. 
+The title is displayed in the menu.  
+```
+---
+id: MyPageID
+title: My new documentation page
+---
+```
+Pages can be grouped in categories in the sidebar.
+The category should be represented as directories in the source tree. 
+
+## Releasing a new version
+Releasing a new version requires you to have yarn locally installed.
+This should only be run by a release manager or on request of the release manager.
+
+Pre-requisites: 
+* The release must have been approved via the release process.
+* The source archive file and signatures must have been uploaded to the Apache locations.
+* The docker convenience images must have been pushed to the Apache docker hub.   
+
+Open a PR and commit all the changes below to the **master** branch, once the auto-build is done, the new documentation will be presented on the website.
+
+### Version the documentation
+Documentation versioning uses the simple **MAJOR.MINOR.PATCH** semver version of the release. 
+This means no letters, release candidate tags or anything like that:  
 ```shell script
 yarn release 0.9.0
 ```
 This command will snapshot all the docs from the current `docs` directory, and copy all files to another new directory
 under `versioned_docs`, e.g `versioned_docs/version-0.9.0`.
 A similar copy will be generated for the sidebar under `versioned_sidebars` that belongs to the documentation version.
 
-Open a PR and commit these changes to the **master** branch, once the auto-build is done, the new documentation will be presented on the website.
+This process will also update the `version.json` file in the root and add the new release as a line item.
 
-## Deploy website
+### Release announcement
+The release announcement is a static Markdown file added to the directory `src/pages/release-announce`.
+The file name is the same a simple semver version of the release:
+``` 
+src/pages/release-announce/0.9.0.md
+```
+The page is in Markdown format and should follow the example of the already existing pages:
+* id and title definition (docusaurus syntax)
+* Apache license
+* Following headings
+  * Title (level 1)
+  * Overview of the release (level 2)
+  * Highlights (level 2)
+    Chosen highlights at level 3 heading
+  * Community update (level 2)
+
+### Update the download page
+The download page contains the link to the Apache source download, signatures etc.
+The table on the page has a specific layout that is correctly rendered after going through the docusaurus build.
+Please do not change the release entry layout in the table.
+
+First step is to update the `latest release` line to the correct version.  
+Second step is to update the table: copy an existing entry and change the links and details to point to the new release.
+All links must be updated:
+* 3 for the download (source tar, checksum and signature)
+* 2 for the docker images (scheduler and admission controller)
```

Comment:
Here should have 3 including the web image.

Reply:
fixed